README.MD

```diff
@@ -22,11 +22,12 @@ It reduces time spent writing and maintaining the same code for different platfo
 📺 [YouTube](https://www.youtube.com/playlist?list=PLlFc5cFwUnmy_oVc9YQzjasSNoAk4hk_C)  
 🤩 [Samples](https://kotlinlang.org/docs/kmm-samples.html)  
 🗄 [Jetpack Compose Components](https://m3.material.io/components)  
-🗄 [Compose Multiplatform Components](https://composables.com)  
+🗄 [Compose Multiplatform Components](https://composables.com)
 🗄 [Compose Material 3 Gallery](https://terrakok.github.io/compose-material-3-gallery/)  
 🎨 [Material Theme Builder](https://materialkolor.com)  
 📚 [Kotlin Multiplatform by Tutorials](https://www.kodeco.com/books/kotlin-multiplatform-by-tutorials)  
 📚 [Simplifying Application Development with Kotlin Multiplatform Mobile](https://www.amazon.com/Simplifying-Application-Development-Kotlin-Multiplatform/dp/1801812586)  
+✏️ [Icons](https://www.composeicons.com)
```

Comment:
I don't see a button to download a vector xml - the default multiplatform icon format

Reply:
What is vector XML? I always used them like ImageVectors in kotlin. Do you have an example so that I can have a look?

Reply:
I mean I would like to download an icon as .xml file and put it to the composeResources directory.
Because svgs don't work on android out of the box. Icons as code slow the build time down

Reply:
I see what you mean. I am currently busy with an other project and I can't promise when/if this will be added. [There is a github issue board](https://github.com/composablehorizons/composeicons/issues/) with request if you would like to request it.

If this is a reason not to merge this, that's alright.